src/Badge.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import { Box, IBox } from './Box';
+
+export enum BadgeVariant {
+  Pill = 'pill',
+  Textual = 'textual',
+}
+
+export enum BadgeColor {
+  Error = 'error',
+  Warning = 'warning',
+  Default = 'default',
+}
+
+const colorMap = {
```

Comment:
Can we move the color definitions into the theme?

Reply:
Done